tests/config.json

```diff
@@ -0,0 +1,77 @@
+{
+  "network": {
```

Comment:
FYI there's also a `cli-config.json` file in the repo which is an older version of this (missing construction API stuff); maybe replace that file with this updated one or just delete that one as well?